presto-spi/src/main/java/com/facebook/presto/spi/connector/ConnectorNodePartitioningProvider.java

```diff
@@ -23,8 +23,24 @@
 import java.util.Map;
 import java.util.function.ToIntFunction;
 
+import static com.facebook.presto.spi.connector.NotPartitionedPartitionHandle.NOT_PARTITIONED;
+import static java.util.Collections.singletonList;
+
 public interface ConnectorNodePartitioningProvider
 {
+    // TODO: Use ConnectorPartitionHandle (instead of int) to represent individual buckets. Currently, it's mixed.
```

Comment:
I meant for this TODO to apply to this class in general. I expanded my coments to make it more explicit.

```
    // TODO: Use ConnectorPartitionHandle (instead of int) to represent individual buckets.
    // Currently, it's mixed. listPartitionHandles used CPartitionHandle whereas the other functions used int.